api/serializers.py

```diff
@@ -0,0 +1,29 @@
+import re
+from rest_framework import serializers
+from greedybear.models import IOC
+from greedybear.consts import REGEX_IP, REGEX_DOMAIN
+
+class IOCSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = IOC
+        exclude = ["related_urls",]
+
+class EnrichmentSerializer(serializers.Serializer):
+    found = serializers.BooleanField(read_only=True, default=False)
+    ioc = IOCSerializer(read_only=True, default=None)
+    query = serializers.CharField(max_length=250)
+
+    def validate(self, data):
```

Comment:
should not be better to just create the custom `validate_query` as Iyou suggested before? @eshaan7 https://www.django-rest-framework.org/api-guide/serializers/#field-level-validation